src/uCodes/ZSortBOSS.cpp

```diff
@@ -605,17 +605,17 @@ void ZSortBOSS_TransformLights( u32 _w0, u32 _w1 )
 		gSP.lights.rgb[i][G] = (f32)(((u8*)DMEM)[(addr+8+1)^3]) * 0.0039215689f;
 		gSP.lights.rgb[i][B] = (f32)(((u8*)DMEM)[(addr+8+2)^3]) * 0.0039215689f;
 
-		gSP.lights.xyz[i][X] = (f32)(((s8*)DMEM)[(addr+16+0)^3]) * 0.00390625f;
```

Comment:
@Gillou68310 There is the same: light vector is signed char. Why you divide it by 256 when convert to float?